storybook/storybook-css/src/docs/typography.stories.mdx

```diff
@@ -23,23 +23,23 @@ In de tabel hieronder vind je deze waardes in pixels:
 
 | niveau      | 0   | 1   | 2   | 3   | 4   | 5   | 6   | 7   |
 | ----------- | --- | --- | --- | --- | --- | --- | --- | --- |
-| min         | 32  | 32  | 28  | 24  | 22  | 20  | 18  | 16  |
-| breakpoint  | 64  | 48  | 36  | 29  | 26  | 23  | 21  | 17  |
-| max         | 80  | 56  | 40  | 32  | 28  | 25  | 22  | 18  |
-| line-height | 1.1 | 1.2 | 1.3 | 1.4 | 1.5 | 1.4 | 1.6 | 1.6 |
+| min         | 32  | 32  | 28  | 24  | 20  | 22  | 18  | 16  |
```

Comment:
This is not correct is it? 5 larger then 4?

Reply:
This is not confusing at all 😬

Reply:
It still is a little bit, but [there's a ticket to look into that](https://gemeente-amsterdam.atlassian.net/browse/DES-244)